README_py.md

```diff
@@ -0,0 +1,70 @@
+# Welcome to a Toy PyBind11 Algorithm
```

Comment:
I think the heading needs to change to something that is more clear on what this document achieves. We can discuss in our meeting.